chef/cookbooks/crowbar/templates/default/crowbar-pre-upgrade.sh.erb

```diff
@@ -54,4 +54,26 @@ ovs-vsctl set-fail-mode <%= bridge %> standalone
 echo "Reloading wicked configuration for all interfaces"
 wicked ifreload all
 
+<% if @compute_node %>
+
+echo "Upgrading services needed to act as nova-compute node"
+
+zypper --non-interactive up openstack-nova-compute
+zypper --non-interactive up openstack-neutron-openvswitch-agent
+zypper --non-interactive up python-pecan
```

Comment:
Last time I spoke with @jgrassler , it seemed like a bad idea for some reason. But indeed it looks strange...